libbeat/tests/system/beat/beat.py

```diff
@@ -283,6 +283,27 @@ def log_contains(self, msg, logfile=None):
         except IOError:
             return False
 
+    def log_contains_count(self, msg, logfile=None):
```

Comment:
The reason I didn't implement it this way is that log_contains can exit much faster then count. So in case we have a large log file it is much more efficient. We discussed in the past doing some tests with large log files, which we don't have yet. So this is probably a premature optimization. Will change it.